client/components/product-card/options.jsx

```diff
@@ -61,7 +55,7 @@ ProductCardOptions.propTypes = {
 			slug: PropTypes.string.isRequired,
 			title: PropTypes.string,
 		} )
-	),
+	).isRequired,
```

Comment:
Are options required now? I can see we have a check for their emptiness at the top of the functional component.

Reply:
Yes, I think the `isRequired` check may be redundant here because of the emptiness check at the top. I'll remove it it.